ping/rust/src/bin/testplan_0440.rs

```diff
@@ -66,9 +66,9 @@ impl PingSwarm for OrphanRuleWorkaround {
 
         while received_pings.len() < number {
             if let Some(SwarmEvent::Behaviour(ping::Event {
-                peer,
-                result: Ok(ping::Success::Ping { .. }),
-            })) = self.0.next().await
+                                                  peer,
+                                                  result: Ok(ping::Success::Ping { .. }),
+                                              })) = self.0.next().await
```

Comment:
Can we undo these formatting changes please?

Reply:
This was resolved with a whole pass of `cargo fmt`